2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1037,6 +1037,58 @@ Questa attività consiste nei seguenti compiti:
 
 = Processi tecnici
 
+== Processo di analisi della missione
+
+_Tailored conformance on ISO/IEC/IEEE 12207:2017 clause 6.4.1 per ISO/IEC/IEEE 12207:2017 Annex A_
+
+=== Scopo
+
+Il processo di analisi della missione definisce i problemi e le opportunità dai quali scaturisce il progetto, caratterizza lo spazio delle soluzioni e determina una classe di soluzione preferita.
+
+=== Strategia di identificazione e analisi della missione
+
+==== Sistemi e servizi abilitanti
+Gli strumenti di comunicazione adottati dal gruppo sono descritti in TODO.
+
+==== Opportunità
+Si analizzano i problemi e le opportunità così da ottenere una visione d'insieme dello status quo. L'obiettivo dell'analisi è la comprensione del contesto e delle cause di problemi e opportunità, e quindi del Capitolato.
```

Comment:
Personalmente, trovo un po' pesante (forse ripetitivo) la locuzione "problemi e le opportunità", che è presente sia nella prima frase che nella seconda. Probabilmente riformulerei in modo più coeso come "Si svolge un'analisi delle problematiche e delle opportunità per acquisire una panoramica completa del contesto presentato dal Capitolato.".